rfc9114.md

```diff
@@ -170,15 +57,15 @@ subsumed by QUIC and describes how HTTP/2 extensions can be ported to HTTP/3.
 
 # Introduction
 
-HTTP semantics ({{RFC9110}}) are used for a broad range of services on the
+HTTP semantics ({{!RFC9110}}) are used for a broad range of services on the
```

Comment:
This won't render correctly.  You need to retain either the reference:

```yaml
normative:
  RFC9110:
    display: HTTP
```

Or use this (which won't work for those names that include a '/', so maybe not).

```suggestion
HTTP semantics ({{!HTTP=RFC9110}}) are used for a broad range of services on the
```